### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ForeverProglamer**

### CODE REVIEW

**Review by vladyslavry141 — APPROVED** [review]
Good job

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 # Mobile Tools for Java (J2ME)
 .mtj.tmp/
+.idea/
```
Comment (VitaliiZZzz) [inline .gitignore:12]: It's ok.

**Review by VitaliiZZzz — COMMENTED** [review]
I need some help from @e-andrew

File: .gitignore (hunk 1 above)
Comment (e-andrew) [reply .gitignore:12]: I can't do anything. Sorry

**Review by e-andrew — APPROVED** [review]

**Pull request merged**